x-pack/plugins/security/public/index.scss

```diff
@@ -1,7 +1,2 @@
-$secFormWidth: 460px;
```

Comment:
__note:__ we use it just in one place right now. We can return this back if we ever decide to generalize this parameter.

Reply:
Yep, this is fine.